### DISCUSSION TITLE
Phase unwrapping and background field removal of 2D data sets

### DISCUSSION THREAD

**Discussion by ogeier** [kschan0214/sepia#91] [Q&A]
Hi, I would like to process 2D EPI data. Is there an option for 2D phase unwrapping and 2D background phase removal? I tried both LAPLACIAN + RESHARP and LAPLACIAN+VSHARP, but there are still slice-to-slice inconsistencies. 
Thanks in advance!

**Comment by kschan0214** [comment]
Hi, unfortunately SEPIA does not support 2D background field removal. If you have STI Suite v3.0, I believe they have a V-SHARP function that supports 2D acquisition but you will need to run that step outside of SEPIA.

**Reply by tsalo** [reply]
Would SEPIA's `VSHARP (STI Suite)` not work?

**Reply by tsalo** [reply]
Actually I just realized that #111 has been merged. Does that mean that SEPIA will now work on 2D acquisitions?
